src/gui.cpp

```diff
@@ -1297,7 +1298,8 @@ void gui::model_browser::render(app& a) {
 			}
 			
 			ImGui::Checkbox("Show Vertex Indices", &_view_params.show_vertex_indices);
-			
+			ImGui::Checkbox("Show Bounding Box", &_view_params.show_bounding_box);
```

Comment:
Not 100% sure if we need/want the bounding box to be renderable, but it was something i used for debugging which was easy enough to turn into a feature. Happy and easy to remove it.

Reply:
The bounding box being visible seems like a nice feature.